lib/full-site-editing/template-parts.php

```diff
@@ -60,6 +60,49 @@ function gutenberg_register_template_part_post_type() {
 }
 add_action( 'init', 'gutenberg_register_template_part_post_type' );
 
+/**
+ * Registers the 'wp_template_section' taxonomy.
+ */
+function gutenberg_register_wp_template_section_taxonomy() {
+	if ( ! gutenberg_is_fse_theme() ) {
+		return;
+	}
+
+	register_taxonomy(
+		'wp_template_section',
```

Comment:
Would it make more sense to use `wp_theme_section` here to align with the other relevant taxonomy we register (`wp_theme`)?

Reply:
Im not sure.  `wp_theme_section` seems odd as its not the section of a theme that is being defined, just the area in a template that the template part is designated for.  `wp_theme` is theme relevant because it links the cpt to the theme and is the theme name/directory, while `wp_theme_section` seems odd because the theme is entirely irrelevant with respect to the section (other than just defining the section for the template parts they supply).

Reply:
^^ That said, im open to what others think makes the most sense.

Reply:
Yes, I'm not sure myself :) Maybe we just need to drop the "template" for `wp_section`, but then there's some higher likelihood of clashes with possible third-party definitions?